src/mono/sample/iOS/Makefile

```diff
@@ -10,6 +10,8 @@ DEPLOY_AND_RUN?=true
 APP_SANDBOX?=false
 STRIP_DEBUG_SYMBOLS?=false # only used when measuring SOD via build-appbundle make target
 HYBRID_GLOBALIZATION?=true
+USE_MONO_RUNTIME?=true
```

Comment:
To double check, should this be true for the currently running scenarios, primarily asking about NativeAOT?

Reply:
NativeAOT is running a different sample `iOS-NativeAOT`. We should unify these samples with the functional tests in the future, since they are identical.